tests/forcing/test_lisflood.py

```diff
@@ -0,0 +1,219 @@
+                                                                          'start_year': 1989},
+                                                               'tasmin': {'end_year': 1999,
+                                                                          'mip': 'day',
+                                                                          'preprocessor': 'daily_temperature',
+                                                                          'start_year': 1989},
+                                                               'tdps': {'end_year': 1999,
+                                                                        'mip': 'Eday',
+                                                                        'preprocessor': 'daily_temperature',
+                                                                        'start_year': 1989},
+                                                               'uas': {'end_year': 1999,
+                                                                       'mip': 'day',
+                                                                       'preprocessor': 'daily_windspeed',
+                                                                       'start_year': 1989},
+                                                               'vas': {'end_year': 1999,
+                                                                       'mip': 'day',
+                                                                       'preprocessor': 'daily_windspeed',
+                                                                       'start_year': 1989}}}},
+            'documentation': {'authors': ['verhoeven_stefan',
+                                          'kalverla_peter',
+                                          'andela_bouwe'],
+                              'projects': ['ewatercycle'],
+                              'references': ['acknow_project']},
+            'preprocessors': {'daily_radiation': {'convert_units': {'units': 'J m-2 '
+                                                                             'day-1'},
+                                                  'custom_order': True,
+                                                  'extract_region': {'end_latitude': 52.2,
+                                                                     'end_longitude': 11.9,
+                                                                     'start_latitude': 46.3,
+                                                                     'start_longitude': 4.1},
+                                                  'extract_shape': {'crop': True,
+                                                                    'method': 'contains',
+                                                                    'shapefile': '/home/verhoes/git/eWaterCycle/ewatercycle/tests/models/data/Rhine/Rhine.shp'},
```

Comment:
The test is failing because of this path. I suggest refactoring as:
```python 
f'{Path(__file__).parents[1]}/models/data/Rhine/Rhine.shp'
```
please implement this for other cases below and add `from pathlib import Path` too.